app/scripts/modules/core/src/cluster/ClusterPostSearchResultSearcher.ts

```diff
@@ -0,0 +1,45 @@
+import { IPromise, IQService } from 'angular';
+
+import {
+  ISearchResultFormatter,
+  searchResultFormatterRegistry
+} from 'core/search/searchResult/searchResultFormatter.registry';
+import { ISearchResultSet } from 'core/search/infrastructure/infrastructureSearch.service';
+import { IClusterSearchResult } from 'core/search/searchResult/model/IClusterSearchResult';
+import { IServerGroupSearchResult } from 'core/search/searchResult/model/IServerGroupSearchResult';
+import { IPostSearchResultSearcher } from 'core/search/searchResult/PostSearchResultSearcherRegistry';
+
+export class ClusterPostSearchResultSearcher implements IPostSearchResultSearcher<IServerGroupSearchResult> {
+
+  private static TYPE = 'clusters';
+
+  constructor(private $q: IQService) {}
+
+  public getPostSearchResults(inputs: IServerGroupSearchResult[]): IPromise<ISearchResultSet[]> {
+
+    const clusters: IClusterSearchResult[] = inputs.map((serverGroup: IServerGroupSearchResult) => {
+      const { account, application, cluster, stack} = serverGroup;
+      return {
+        account,
+        application,
+        cluster,
+        displayName: cluster,
+        href: `#/applications/${application}/${ClusterPostSearchResultSearcher.TYPE}?acct=${account}&q=cluster:${cluster}&stack=${stack}`,
+        provider: serverGroup.provider,
+        stack,
+        type: ClusterPostSearchResultSearcher.TYPE,
+        url: `/applications/${application}/${ClusterPostSearchResultSearcher.TYPE}/${account}/${cluster}`
```

Comment:
what's the difference between `href` and `url`?

Reply:
So that's on me.  I never looked at why we get both, but I did just now.  It looks like Clouddriver sets the `url` via `urlMappings` in `CatsSearchProvider` and the `href` property is set in the `InfrastructureSearcher` class (both legacy functionality).  Cam wasn't terribly wild about having that `url` property being set in Clouddriver.  Looks like that functionality predates July 2015 so I'm guessing the original reasoning behind it has probably been forgotten since those commit dates were moving code around.  After an ~exhaustive~ cursory search of the UI code, it doesn't appear the `url` property is used anymore (I assume it did at one point).  We could probably dump this in Clouddriver and mark it as a BC.